src/main/java/org/sonarlint/intellij/ui/filter/FindingsFilter.kt

```diff
@@ -70,7 +71,10 @@ data class FilteredFindings(
 class FindingsFilter(private val project: Project) {
 
     fun filterAllFindings(file: VirtualFile?, criteria: FilterCriteria): FilteredFindings {
-        val rawFindings = loadRawFindings(file)
+        val rawFindings = when (criteria.scopeMode) {
+            ScopeMode.CURRENT_FILE -> loadRawFindings(file)
+            ScopeMode.OPEN_FILES -> loadRawFindingsForOpenFiles()
```

Comment:
The "mode" is called OPEN_FILES, but in the UI the filter is 'All files'. Should we align both?